pkg/operator/ceph/csi/spec.go

```diff
@@ -174,6 +174,9 @@ const (
 	// default log level for csi containers
 	defaultLogLevel uint8 = 0
 
+	// default provisioner replicas
+	defaultProvisionerReplicas uint8 = 2
```

Comment:
Why must this be a uint8?

Reply:
it can be any type, I used this one as it does not go for a higher number (max 255).

Reply:
any suggestion?

Reply:
I would use the same type as used by the replica count we will give to kubernetes (int32).

```go
type DeploymentSpec struct {
    // Number of desired pods. This is a pointer to distinguish between explicit
    // zero and not specified. Defaults to 1.
    // +optional
    Replicas *int32 `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`

    // Label selector for pods. Existing ReplicaSets whose pods are
    // selected by this will be the ones affected by this deployment.
    // It must match the pod template's labels.
    Selector *metav1.LabelSelector `json:"selector" protobuf:"bytes,2,opt,name=selector"`
    // ....
```